crates/client/src/client/schema/chain.rs

```diff
@@ -4,134 +4,234 @@ use crate::client::schema::{
     U32,
     U64,
 };
-use fuel_core_types::fuel_tx::ConsensusParameters as TxConsensusParameters;
 
 #[derive(cynic::QueryFragment, Debug)]
 #[cynic(schema_path = "./assets/schema.sdl")]
 pub struct ConsensusParameters {
-    pub contract_max_size: U64,
+    pub tx_params: TxParameters,
+    pub predicate_params: PredicateParameters,
+    pub script_params: ScriptParameters,
+    pub contract_params: ContractParameters,
+    pub fee_params: FeeParameters,
+    pub chain_id: U64,
+    pub gas_costs: GasCosts,
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct TxParameters {
     pub max_inputs: U64,
     pub max_outputs: U64,
     pub max_witnesses: U64,
     pub max_gas_per_tx: U64,
-    pub max_script_length: U64,
-    pub max_script_data_length: U64,
-    pub max_storage_slots: U64,
+}
+
+impl From<TxParameters> for fuel_core_types::fuel_tx::TxParameters {
+    fn from(params: TxParameters) -> Self {
+        Self {
+            max_inputs: params.max_inputs.0,
+            max_outputs: params.max_outputs.0,
+            max_witnesses: params.max_witnesses.0,
+            max_gas_per_tx: params.max_gas_per_tx.0,
```

Comment:
Could you use `into()` please, instead of `0` in all places?=)
```suggestion
            max_inputs: params.max_inputs.into(),
            max_outputs: params.max_outputs.into(),
            max_witnesses: params.max_witnesses.into(),
            max_gas_per_tx: params.max_gas_per_tx.into(),
```